### ISSUE TITLE
Correct range of input values for MobileNet

### ISSUE THREAD

**Issue by ogreyesp** [tensorflow/tensorflow#47465]
Hello,

I want to use the implementation of MobileNetV3 (either the MobileNet_large or MobileNet_small version) in my project. I'm using the TensorFlow v2.4.1.

 I get confused when I read the documentation about the range of input values for MobileNet-based models. According to this [link](https://tfhub.dev/google/imagenet/mobilenet_v3_large_100_224/classification/5)  the expected range of values for the model is [0,1]. However, according to this other [link](https://www.tensorflow.org/tutorials/images/transfer_learning), the correct range of values is [-1, 1]. A similar issue was already reported at this [link](https://github.com/tensorflow/hub/issues/637).

On the other hand, to my surprise, the function preprocess_input of the module [MobileNetV3](https://github.com/tensorflow/tensorflow/blob/v2.4.1/tensorflow/python/keras/applications/mobilenet_v3.py#L556-L558) does not apply any change over the input; i.e. the function is returning directly the input.

```
@keras_export('keras.applications.mobilenet_v3.preprocess_input')
def preprocess_input(x, data_format=None):  # pylint: disable=unused-argument
  return x

```

So, what should be the correct range of input values for MobileNetV3?

**Comment by sayakpaul** [comment]
Hi @ogreyesp. The following options might be worth trying while someone from the team replies to your query - 

* Load up the MobileNetV3 model and print the summary of the model. Inspect the first few layers and if you see something like Normalization or Rescaling to be present there then you can be sure that you can pass an input image as is. This is in fact the case for all the EfficientNet models that are present in `tf.keras.applications`. 
* If the above option is not applicable then maybe try with both [0, 1] and [-1, 1] range scaling and see which one works. I know this sounds terrible, so apologies in advance for this suggestion.

**Comment by ogreyesp** [comment]
Hi @sayakpaul. Thank you for your suggestions, they were very helpful.

Indeed, MobileNet v3 has a rescaling layer, so we don't need to rescale the images before passing them to the model.

Thanks!

**Comment by ymodak** [comment]
For completeness of the issue :
 `tf.keras.applications.MobileNet` (v1,v2,v3) models expect pixel [range [-1,1]](https://www.tensorflow.org/api_docs/python/tf/keras/applications/mobilenet_v3/preprocess_input#returns_1) where as TF Hub image models expect [0,1] range. Thanks!

**Comment by ogreyesp** [comment]
Hi @ymodak 

I'm not sure... according to the implementation of MobileNetV3 at this [link](https://github.com/tensorflow/tensorflow/blob/v2.4.1/tensorflow/python/keras/applications/mobilenet_v3.py#L561-L563), more specifically line 265, the input is rescaled to [0,1] scale.

```
  x = img_input
  x = layers.Rescaling(1. / 255.)(x) 
```

**Comment by ymodak** [comment]
The docs are updated for MobileNetV3 https://www.tensorflow.org/api_docs/python/tf/keras/applications/mobilenet_v3/preprocess_input
